modules/apigee_edge_teams/src/Entity/Controller/TeamController.php

```diff
@@ -233,4 +235,39 @@ public function delete(string $entity_id): EntityInterface {
     return $entity;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getEntities(PagerInterface $pager = NULL, string $key_provider = 'id'): array {
+    if ($this->orgController->isOrganizationApigeeX()) {
+      // Getting the channelId & filter enable check from Config form.
+      $channelconfig = \Drupal::config('apigee_edge_teams.team_settings');
+      $channelid = $channelconfig->get('channelid');
+      $channelfilter = $channelconfig->get('enablefilter');
+
+      if ($channelfilter) {
+        if ($channelid) {
+          $queryparam = [
+            'filter' => 'channelId=' . $channelid,
+          ];
+        }
+        else {
+          $defaultChannelId = TeamAliasForm::originalChannelId();
+          $queryparam = [
+            'filter' => 'channelId=' . $defaultChannelId,
+          ];
+        }
```

Comment:
Use ternary operator instead to reduce code to fewer lines.

Reply:
Hi @shishir-intelli  The required changes are made.